### PR TITLE
Remove many instances where you are forced to fight friendly governments

### PR DESCRIPTION
**Pull request by PeacefulPotato**
**Content (Artwork / Missions / Jobs)**

## Summary
I updated a lot of missions so that you aren't forced to fight friendly governments, and added a few pirate jobs where you have to fight bounty hunters.

## Save File
[Test Pilot.txt](https://github.com/endless-sky/endless-sky/files/8056931/Test.Pilot.txt)

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}

### CODE REVIEW

**Comment by PeacefulPotato** [comment]
Fair enough. I removed the separate faction.

**Comment by EjoThims** [comment]
This makes a lot more sense now imo.

File: data/human/boarding missions.txt [hunk 1]
```diff
@@ -369,16 +373,33 @@ mission "Pirate Ambush"
...
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		"combat rating" > 100
+		random < 5
+	source
+		government "Pirate (Rival)"
```
[the remark this answers is not in the document]
Comment (PeacefulPotato) [reply data/human/boarding missions.txt:393]: Oops, I may have replaced the wrong part.

File: data/human/pirate jobs.txt [hunk 2]
```diff
@@ -2217,6 +2217,50 @@ mission "Escort Stolen Vessel (Extra Large, South, Dangerous Path)"
 	on visit
 		dialog phrase "generic arrived-without-npc dialog"
 
+mission "Eliminating Law Enforcement (Bounty, Small)"
+	name "Bounty Hunters near <system>"
+	repeat
+	job
+	description "A Bounty Hunter fleet near the <system> system has halted our raids. Destroy them and return to <planet> for a payment of <payment>."
+	to offer
+		random < 25
```
[the remark this answers is not in the document]
Comment (PeacefulPotato) [reply data/human/pirate jobs.txt:2226]: What do you mean?

File: data/governments.txt [hunk 3]
```diff
@@ -436,6 +437,26 @@ government "Pirate"
...
+	"fine" 0
+	"friendly hail" "friendly pirate"
+	"hostile hail" "hostile pirate"
+	raid "pirate raid"
+
+government "Pirate (Rival)"
+	swizzle 6
+	color .78 0 0
+	
+	"player reputation" -10
```
[the remark this answers is not in the document]
Comment (PeacefulPotato) [reply data/governments.txt:452]: I thought I put one in. My bad.

File: data/human/boarding missions.txt [hunk 4]
```diff
@@ -8,40 +8,40 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
-conversation "assisting merchant"
+conversation "assisting friendly ship"
 	`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment>.`
 		decline
 
-mission "Assisting Merchant (Small)"
+mission "Assisting Friendly Ship (Small)"
```
[the remark this answers is not in the document]
Comment (PeacefulPotato) [reply data/human/boarding missions.txt:15]: All right. I added a comment instead explaining that it says merchant for compatibility purposes.

**Review by Amazinite — CHANGES_REQUESTED** [review]
The Pirate and Pirate (Rival) governments should not be enemies with one another, as then you could just let the naturally spawning pirates finish the mission for you. I think that the Pirate (Rival) government should also have a bribe of 0, although I wouldn't consider that change necessary.

The original purpose behind having the job to attack rival pirates was to have them start out friendly and then require the player to attack them; it's not as if they know that you're coming for them. So I think that we should set the rival government to the main government's reputation when the mission is accepted.
```
"reputation: Pirate (Rival)" = "reputation: Pirate"
```

Please remove the changes to the boarding missions file. I don't see pirates giving the player credits for assisting them at the same rate as merchants, and the purpose of the boarding pirate mission was like finding treasure on a pirate ship, so it would seem weird to me for this mission to spawn on merchants.

I also think we could go without the changes to the FW file; in both cases you are fighting pirates, so I don't see why we'd change the governments up. At most I'd keep the changes from pirate to Alpha, but the pirate to bounty change should definitely be reverted.

I'd also say remove the addition of the new jobs as that's a separate issue from what this PR's main intent is dealing with governments that the player fights.

**Comment by PeacefulPotato** [comment]
I will change the rival government so that it is more like the disguised merchants.
The boarding missions file was because it felt weird that merchants would "thank you" for pillaging them, or that you would "raid" a pirate ship you repaired. I didn't realize there was a credit difference, so I could split up the missions based on whether or not the pirates/merchants are friendly. As for the FW missions, I will keep the Alphas but I'm fine with changing the Ryk Bartlett missions back to pirates, since now that I think about it this isn't exactly a bounty. The problem is as you said other friendly ships could do the mission for you. I will also move the new jobs away and possibly add it later as another pull request if I feel up to it.

File: data/human/pirate jobs.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (PeacefulPotato) [reply data/human/pirate jobs.txt]: OK, doesn't matter since I reverted the changes.

**Review by Amazinite — APPROVED** [review]

**Pull request merged**